CapMonsterCloud.Client/CapMonsterCloudClientFactory.cs

```diff
@@ -47,7 +47,7 @@ public ICapMonsterCloudClient Create()
         public static ICapMonsterCloudClient Create(ClientOptions options)
             => Create(options, httpMessageHandlerFactory: null, configureClient: null);
 
-        private static ICapMonsterCloudClient Create(
+        internal static ICapMonsterCloudClient Create(
```

Comment:
can be leave private. Use CapMonsterCloudClientFactory ctor

Reply:
done